R/data.R

```diff
@@ -97,6 +97,14 @@ NULL
 #' @format NULL
 "growth_rates_floodplain"
 
+#' Prey Density
+#' Vector defining prey density level ("low", "med", "hi", "max") by watershed
+"prey_density"
```

Comment:
Is there a default that prey levels are set at while running the model? That might be helpful to add